doc/decisions/simplify_api.md

```diff
@@ -48,6 +48,82 @@ However, the API changes follow these rules:
 - _Remove_ all functions that can be replaced by others (e.g. `ksPop`)
 - _Rename_ all functions to start with `elektra` as described in [Elektra Prefix](elektra_prefix.md), and use `KeySet` instead of `ks` (also applies to names above)
 
+Afterwards the Elektra Core Public API would look as follows:
+
+#### KDB
+
+- kdbOpen
+- kdbClose
+- kdbGet
+- kdbSet
+
+#### Key
+
+- keyNew
+- keyVNew
+- keyCopy
+- keyClear
+- keyDel
+- keyIncRef
+- keyDecRef
+- keyGetRef
+- keyLock
+- keyIsLocked
+- keyDup
+
+#### Meta
+
+- keyMeta
+
+#### Tests
+
+- keyCmp
+- keyNeedSync
+- keyIsBelow
+- keyIsBelowOrSame
+- keyIsDirectlyBelow
+
+#### Name
+
+- keyName
+- keySetName
+- keyAddName
+- keyNameSize
+- keyUnescapedName
+- keyUnescapedNameSize
+- keyBaseName
+- keySetBaseName
+- keyAddBaseName
+- keyBaseNameSize
+- keyGetNamespace
+- keySetNamespace
+
+#### Value
+
+- keyValue
+- keyValueSize
+- ketSetValue
+- keyString
```

Comment:
What does this function do?

Reply:
Good question, I would have thought this might be a convenience method to return the value interpreted as string, but this just opens up more questions about encoding and stuff. This might be solved by adding an encoding parameter but this just complicates things further. Maybe it's just a bad idea.

Reply:
Such a function might make sense, but shouldn't be part of the core, but one of the extra libraries (#4245).